src/SampleFormat.h

```diff
@@ -125,18 +129,29 @@ class GrowableSampleBuffer : private SampleBuffer
 //
 
 AUDACITY_DLL_API
-void      CopySamples(constSamplePtr src, sampleFormat srcFormat,
-                      samplePtr dst, sampleFormat dstFormat,
-                      unsigned int len, bool highQuality=true,
-                      unsigned int srcStride=1,
-                      unsigned int dstStride=1);
+//! Copy samples from any format into the widest format, which is 32 bit float, with no dithering
+/*!
+ @param src address of source samples
+ @param srcFormat format of source samples, determines sizeof each one
+ @param dst address of floating-point numbers
+ @param len count of samples to copy
```

Comment:
The reason for stride (I presume for stereo) is worth mentioning here.

Reply:
So I should use the word "interleaving" in comments.